### DISCUSSION TITLE
API down 21 settembre ?

### DISCUSSION THREAD

**Discussion by Psicogest** [fattureincloud/api#353] [Q&A] [answered]
Buongiorno,

dalle prime ore del giorno 21 settembre le chiamate alle API vanno in timeout, 504 gateway timeout.
Sto provando anche dal client integrato nella documentazione e ottengo errore ugualmente.
Ci sono tempi previsti per il ritorno online ?

grazie

AdM

**Comment by valmoz** [comment]
Buongiorno @Psicogest,
Per questo genere di richieste vi prego gentilmente di contattare il supporto sulla pagina principale di Fatture in Cloud, che si occuperà di avviare tutte le procedure del caso.

Per quanto mi riguarda provvederò a passare la sua segnalazione al supporto in via informale, al momento non mi sono giunte altre segnalazioni a riguardo.

Grazie
Mauro

**Reply by valmoz** [reply]
Confrontandomi con i colleghi risulta essere presente il disservizio da lei segnalato. La pregerei comunque di contattare il supporto clienti per restare aggiornato sull'evoluzione del problema.
I colleghi stanno già lavorando per riportare la soluzione alla normalità.

Grazie
Mauro

**Comment by dmigliacci** [comment]
Buongiorno, confermo di avere lo stesso problema in corso.
Che blocca il flusso di lavoro.

Aspettiamo news, intanto scriviamo a FIC supporto

**Comment by valmoz** [comment]
Buongiorno @dmigliacci @Psicogest la situazione dovrebbe essere tornata alla normalità, confermate?

**Reply by Psicogest** [reply]
a me va ancora in timeout.
Riprovo fra un 10-15 minuti
grazie
AdM

**Reply by DanieleMancini** [reply]
idem riceviamo ancora timeout.

**Reply by valmoz** [reply]
Ok, segnalo nuovamente. Grazie

**Reply by Psicogest** [reply]
Poco fa (11:05 circa) la chiamata è andata a buon fine, cmq tengo d'occhio per vedere se capita nuovamente
grazie
AdM

**Reply by valmoz** [reply, marked as answer]
Perfetto @Psicogest 
chiedo anche a @DanieleMancini e @dmigliacci conferma della risoluzione del problema.

Mauro

**Reply by DanieleMancini** [reply]
anche noi 11:06 chiamata andata a buon fine.

**Reply by dmigliacci** [reply]
confermo sembra rientrato, thx
